ledger-core/v2/virtual/object/object.go

```diff
@@ -14,6 +14,9 @@ import (
 	"github.com/insolar/assured-ledger/ledger-core/v2/virtual/descriptor"
 )
 
+// Stores information about Executor (key) and its pending execution requests (value)
```

Comment:
I don't think it's proper way to organise this table. It would be hard to do de-duplication checks. I, also, think that this is out of scope for this task.

Reply:
Remove this queue from this PR